samples/requirements.txt

```diff
@@ -1,4 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release
```

Comment:
how are you going to release this file?
I mean once we have created OV GenAI release packaged, we cannot change this file.

Reply:
After OpenVINO is released this line can be removed and GenAI is released after that.